R/pkg/R/functions.R

```diff
@@ -1250,7 +1250,7 @@ setMethod("upper",
 #'}
 setMethod("var",
           signature(x = "Column"),
-          function(x, y = NULL, na.rm = FALSE, use) {
+          function(x) {
```

Comment:
Correct, since the generic is set properly.
There's a test in test_sparkSQL.R for stats::var with the y param
```
> var(1:5, 1:5)
[1] 2.5
> var(1:5, 1:5, na.rm=T)
[1] 2.5
```
Extra param `y`, `na.rm` can still be passed when calling `var` on a Column but they are ignored (just as before)
I discover this while working on #9785 for `lag` https://github.com/apache/spark/pull/9785/files#diff-d97f9adc2dcac0703568c799ff106987